ci/docker_ci.sh

```diff
@@ -12,12 +12,12 @@ ENVOY_DOCKER_IMAGE_DIRECTORY="${ENVOY_DOCKER_IMAGE_DIRECTORY:-${BUILD_STAGINGDIR
 
 # Setting environments for buildx tools
 config_env() {
-  # Qemu configurations
-  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+  # Install QEMU emulators
+  docker run --rm --privileged tonistiigi/binfmt --install all
 
   # Remove older build instance
   docker buildx rm multi-builder || :
-  docker buildx create --use --name multi-builder --platform linux/arm64,linux/amd64
+  docker buildx create --use --name multi-builder
```

Comment:
does this not stop it from building for arm ?

Reply:
platforms can be autodetected - https://github.com/docker/buildx#--platform-platforms

https://github.com/envoyproxy/envoy/blob/3bddd1a8f8f67174d487a06c690f67b944b9e0da/ci/docker_ci.sh#L72 does the `buildx build` for the `ARCH` specified

Reply:
> platforms can be autodetected

lets keep this explicit

Reply:
ok

Reply:
fwiw manually setting it is not recommended unless you know what you are doing. This will override the detection meaning that if you don't actually have a proper setup build will still be attempted and will fail with some `exec format error` deeper in the stack. The main case where this flag is used is to set arch priorities for multi-node build clusters but that does not seem to be your use-case.

Reply:
👍  thanks for clarifying :)

Reply:
@arkodg do you want to create a pr to update ?

Reply:
> This will override the detection meaning that if you don't actually have a proper setup build will still be attempted and will fail with some exec format 

which in a way we want as long as it fails so the setup can be fixed - these *are* the architectures we want and only want

not sure if the combination of doing `--install all` and not specifying this means it would create lots of images that dont get published, but lets see

Reply:
yah `--install all` would setup emulators for all supported architectures, not build all images, that is governed by the `buildx bake` cmd
If we want to have a more cautious and a fail fast and early approach, we should only `install` for `linux/arm64`

Reply:
i see